main.go

```diff
@@ -45,6 +45,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
 	infrav1alpha4 "github.com/nutanix-cloud-native/cluster-api-provider-nutanix/api/v1alpha4"
+	infrastructurev1beta1 "github.com/nutanix-cloud-native/cluster-api-provider-nutanix/api/v1beta1"
```

Comment:
Isn't this redefining the same import as infrav1beta1?

Reply:
kubebuilder now uses complete name of the package with latest version. keeping latest and removed old duplicate